lib/opensusebasetest.pm

```diff
@@ -456,6 +457,8 @@ sub wait_boot {
 
     # on s390x svirt encryption is unlocked with workaround_type_encrypted_passphrase before here
     unlock_if_encrypted if !get_var('S390_ZKVM');
+    # On remote backends we sync on Welcome message already where possible, mitigating bsc#1112109
+    wait_serial(get_login_message()) unless is_remote_backend;
```

Comment:
Oh, so this is why we have in so many tests this failing step. The return value is ignored. Does this wait_serial actually work in your local tests?

Reply:
Now I have actually noticed that it failed on ppc64. I will create new PR to remove this one and addressing you comment below, as reverting export changes will create many conflicts.